tests/test_uniswapv2_market_maker_keeper.py

```diff
@@ -400,7 +510,7 @@ def test_should_remove_dai_eth_liquidity(self):
         keeper.testing_feed_price = True
         keeper.test_price = Wad.from_number(PRICES.ETH_DAI_REMOVE_LIQUIDITY.value)
 
-        time.sleep(10)
+        time.sleep(25)
```

Comment:
This can be reduced back down to 10 seconds, but ultimately, the sleeps are needed to provide time for the keeper to see that it should remove liquidity, submit the transaction, wait for response, and update state. This is a full integration test running against a full local blockchain.

Reply:
Ganache offers `evm_increaseTime` to speed things up if blocks need to pass; if helpful, [here's an example](https://github.com/makerdao/pymaker/blob/master/tests/helpers.py#L38).  Generally, 10 seconds seems okay.

Reply:
Oh awesome, thanks! Yeah probably not sustainable at scale. I will update to use that

Reply:
Per offline conversation, this wouldn't fit the use case. Leaving as is.